### DISCUSSION TITLE
https apis nocodb com tag DB Table Row [Oct 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6646] [Discord Threads]
> 👨‍🦲 **dstala**

**Comment by o1lab** [comment]
> 🙋‍♂️ **iqbalpb**  
#### Thanks <@857998597474484295>  🤝  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1147749040826634312)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
